nucypher/cli/commands/deploy.py

```diff
@@ -368,15 +377,12 @@ def upgrade(general_config, actor_options, retarget, target_address, ignore_depl
 @group_general_config
 @group_actor_options
 def rollback(general_config, actor_options):
-    """
-    Rollback a proxy contract's target.
-    """
+    """Rollback a proxy contract's target."""
     emitter = general_config.emitter
     ADMINISTRATOR, _, _, _ = actor_options.create_actor(emitter)
-
     if not actor_options.contract_name:
         raise click.BadArgumentUsage(message="--contract-name is required when using --rollback")
-    ADMINISTRATOR.rollback_contract(contract_name=actor_options.contract_name)
+    return ADMINISTRATOR.rollback_contract(contract_name=actor_options.contract_name)
```

Comment:
I know the goal is to not change logic oo much but should we paint the receipts summary here instead of returning them?

Reply:
Yes - in a follow up PR.

Reply:
:heavy_check_mark:  - paint rollback receipts.